how-to/customize-workspace/client/src/framework/shapes/platform-shapes.ts

```diff
@@ -1,3 +1,13 @@
+export interface PlatformLocale {
+	/**
+	 * A string with a [BCP 47 language tag](http://tools.ietf.org/html/rfc5646).
+	 *  For the general form and interpretation of the locales argument,
+	 *  see the [`Intl` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_identification_and_negotiation)
+	 */
+	tag: string;
+	/** An [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/Locale#Parameters) with some or all of options of the locale. */
+	options?: Intl.LocaleOptions;
```

Comment:
Not sure we should be using Intl.LocaleOptions , did you see all the properties in the schema, should probably be more explicit

Reply:
It was more something that could then be passed to an new Intl.Locale(tag, options) for use if they wanted to. All of the settings are optional.

Reply:
Removing locale from this pull request as it was an add on and should go through it's own pull request/discussion.